### ISSUE TITLE
[BUG] Controller: bml_conn_map doesn't display GW_MASTER mac appropriately

### ISSUE THREAD

**Issue by adam1985d** [prplfoundation/prplMesh#816]
## Describe the bug

seen on RAX40 and Turris Omnia (not check on other platforms)

bml_disconnect: return value is: BML_RET_OK, Success status
Output of bml_nw_map_query:
bml_connect: return value is: BML_RET_OK, Success status
bml_nw_map_query: return value is: BML_RET_OK, Success status
Name: GW_MASTER, Type: GW (1), State: Connected (2), **MAC: º¿•\¯À¨x,** IP: 192.168.1.120, Backhaul: 
  Radio[0]: Interface: wlan0, Vendor: Intel, Channel: 1, Bandwidth: 20, CAC Completed: 1, AP Active: true, Driver Version: NA
    VAP[0]: BSSID: 02:9a:96:fb:59:11, SSID: prplMesh
    VAP[1]: BSSID: 02:9a:96:fb:59:12, SSID: wave_11
    VAP[2]: BSSID: 02:9a:96:fb:59:13, SSID: wave_12
    VAP[3]: BSSID: 02:9a:96:fb:59:14, SSID: wave_13
  Radio[1]: Interface: wlan2, Vendor: Intel, Channel: 116, Bandwidth: 40, CAC Completed: 1, AP Active: true, Driver Version: NA
    VAP[0]: BSSID: 02:9a:96:fb:59:22, SSID: prplMesh
    VAP[1]: BSSID: 02:9a:96:fb:59:23, SSID: wave_43
    VAP[2]: BSSID: 02:9a:96:fb:59:24, SSID: wave_44
    VAP[3]: BSSID: 02:9a:96:fb:59:25, SSID: wave_45

## Steps to reproduce

/opt/prplmesh/bin/beerocks_cli -c bml_conn_map

## Expected behavior

MAC should be displayed appropriately

**Comment by arnout** [comment]
Works for me in docker:

```
$ docker exec gateway-arnout $PWD/build/install/bin/beerocks_cli -c bml_conn_map
bml_connect: return value is: BML_RET_OK, Success status
bml_nw_map_query: return value is: BML_RET_OK, Success status

GW_BRIDGE: name: GW_MASTER, mac: 00:11:22:33:00:00, ipv4: 10.89.1.67
    ETHERNET: mac: 00:11:22:33:00:01
        IRE_BACKHAUL: mac: aa:bb:cc:11:00:30, ch: 0, bw: 160H
```

**Comment by adam1985d** [comment]
> Works for me in docker:
> 
> ```
> $ docker exec gateway-arnout $PWD/build/install/bin/beerocks_cli -c bml_conn_map
> bml_connect: return value is: BML_RET_OK, Success status
> bml_nw_map_query: return value is: BML_RET_OK, Success status
> 
> GW_BRIDGE: name: GW_MASTER, mac: 00:11:22:33:00:00, ipv4: 10.89.1.67
>     ETHERNET: mac: 00:11:22:33:00:01
>         IRE_BACKHAUL: mac: aa:bb:cc:11:00:30, ch: 0, bw: 160H
> ```

I noticed the issue on the CI printouts - perhaps this is only a CI issue or has been resolved?

I will check and will handle accordingly

**Comment by arnout** [comment]
The issue was introduced by commit 74d8454506d692d9feba3f697a0c28ff63c85a8a. That commit removed the redundant mac_to_string() calls for for printing out sMacAddr structs. But the addresses in BML_NODE are not sMacAddr, but open-coded `uint8_t[6]`. So the mac_to_string is still needed.

**Comment by ghost** [comment]
Fixed by #1431
